react/features/prejoin/functions.js

```diff
@@ -36,6 +36,16 @@ export function isDisplayNameRequired(state: Object): boolean {
         || state['features/base/config'].requireDisplayName;
 }
 
+/**
+ * Selector for determining if the display name from prejoin page is read only.
+ *
+ * @param {Object} state - The state of the app.
+ * @returns {boolean}
+ */
+export function isPrejoinNameReadOnly(state: Object): boolean {
+    return Boolean(state['features/base/jwt']?.jwt?.name);
```

Comment:
`jwt?.user?.name?   || jwt?.user?.email` ? 
think it sets either name or email right? also both are under 'user' not directly in jwt I think

Reply:
Right. Good catch! I have not tested enough. Fixed. We are using only the `name` here not the `email`